model.js

```diff
@@ -10,4 +10,11 @@ exports.fetchCategories = () => {
   });
 };
 
+exports.fetchReviewsById = (inputId) => {
+  const queryString = `SELECT * FROM reviews WHERE review_id = ${inputId};`;
```

Comment:
MUST sanitise this - should not use template literals here. Make use of pg-format to replace `...review_id = $1`, [inputId]. Because you've used a variable of queryString, you can pass the array of inputId in the return db.query(queryString, [inputId]) - this should act in the same way as if you'd done it all in the query string.